neuraxle/steps/numpy.py

```diff
@@ -346,7 +346,8 @@ class ToNumpy(ForceHandleMixin, BaseStep):
     Convert data inputs, and expected outputs to a numpy array.
     """
 
-    def _will_process(self, data_container: DataContainer, context: ExecutionContext) -> (DataContainer, ExecutionContext):
+    def _will_process(self, data_container: DataContainer, context: ExecutionContext) -> (
+            DataContainer, ExecutionContext):
```

Comment:
I'd like to see one big PR to uniformize everything using the most default formatter of PyCharm

Reply:
Created #339